### DISCUSSION TITLE
Working with view resources

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#1532] [Help]
First of all, thank you for such a wonderful job. I'm new here and trying to create a menu management resource. How can I get such a structure in view resource? I mean the layout, not the whole code. I hope I explained it normally )

For example:

![download](https://user-images.githubusercontent.com/29732308/153268090-53ffc188-4227-4a10-b9c8-ed9a859aca01.png)

**Comment by ryangjchandler** [comment]
@ArtMin96 This is something that you'll have to build yourself. You can use a `View` field type and render your own view in the form.

**Reply by danharrin** [reply]
I'd suggest just using a custom page for this IG? Depending if your menu items are separate Eloquent records or not.

**Reply by ArtMin96** [reply]
Yes, menu items are separate Eloquent records. And which is the right option, to do everything with Custom pages?
Or can I use the custom page inside the view resource?

**Reply by danharrin** [reply]
Yeah, I'd say just build a custom page for this? Full flexibility.

**Reply by ArtMin96** [reply]
Ok. Thanks.
